File: DayOne/Java/LevelOne/NonPerfectSquareArrayTest.java
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

public class NonPerfectSquareArrayTest{

    @Test
    public void testThatIHaveAnArrayAndIReplaceElementsofArrayThatArePerfectSquaresWithMiusOne(){

    int [] array = {4, 7, 9, 10, 49, 6};

    double [] arrayCheck = {4.0,-1.0 ,9.0, -1.0, 49.0, -1.0};

    double [] actual = NonPerfectSquareArray.nonPerfectSquareCheck(array);

    assertArrayEquals(actual,arrayCheck);
    }

}

File: DayOne/Python/LevelOne/non_perfect_square_array_test.py
from unittest import TestCase

from non_perfect_square_array import *

class TestPerfectSquareArray(TestCase):

    def test_that_i_have_an_array_and_replace_elements_that_are_perfect_squares_with_minus_one(self):

        array =  [4, 7, 9, 10, 49, 6]

        array_check =  [4, -1, 9, -1, 49, -1]

        self.assertEqual(array_check,non_perfect_square_check(array))

        
